libs/core/kiln_ai/adapters/extractors/litellm_extractor.py

```diff
@@ -20,6 +21,16 @@
 from kiln_ai.utils.litellm import get_litellm_provider_info
 from kiln_ai.utils.pdf_utils import split_pdf_into_pages
 
+
+def max_pdf_page_concurrency_for_model(model_name: str) -> int:
```

Comment:
This should be moved into our model_config, no hard coded checks for model names. Can do it next release.